mdbook testが失敗する

https://app.circleci.com/pipelines/github/rust-lang-ja/atcoder-rust-resources/58/workflows/4a071de3-64a2-46c5-8143-4b9ba7472bd3/jobs/58

```console
2020-04-06 14:55:19 [ERROR] (mdbook::utils): Error: Rustdoc returned an error: 
running 1 test
test /tmp/mdbook-jyyVoP/participate/before-contest/generate-project.md - プロジェクトを生成する::cargo_generate (line 36) ... FAILED

failures:

---- /tmp/mdbook-jyyVoP/participate/before-contest/generate-project.md - プロジェクトを生成する::cargo_generate (line 36) stdout ----
error: expected expression, found `$`
 --> /tmp/mdbook-jyyVoP/participate/before-contest/generate-project.md:37:1
  |
3 | $ cargo init abc000
  | ^ expected expression

error: aborting due to previous error
```

--- a/src/participate/before-contest/generate-project.md
+++ b/src/participate/before-contest/generate-project.md
@@ -33,7 +33,7 @@ Done! New project created /path/to/project/abc000
 
 まずは空のプロジェクトを作成します。
 
-```
+```console
 $ cargo init abc000
      Created binary (application) package
 ```

--- a/src/participate/during-contest/write-tests.md
+++ b/src/participate/during-contest/write-tests.md
@@ -8,7 +8,7 @@ Rust には組み込みでテストをするための機構が備わっていま
 
 なんとなく見れば分かるように、 `output_with_stdin()` の引数にサンプル入力を渡し `assert_eq!(output.stdout_str(), <expected output>);` で期待する出力と一致するかどうかを確かめます。一致しない場合、テストは失敗します。
 
-```rust
+```rust,ignore
 use cli_test_dir::*;
 
 const BIN: &'static str = "./main";

